### PR TITLE
Statistics Documentation

### PR DESCRIPTION
**Pull request by RickiJay-WMDE**

### CODE REVIEW

File: docs/data-list.md [hunk 1]
```diff
@@ -715,13 +810,54 @@ Data abbreviated for brevity.
...
+        "wikibaseCount": 1,
+        "year": 2011
+      },
+      ...
+    ]
+  }
+}
+```
+
+Data abbreviated for brevity.
```
Comment (rti) [inline docs/data-list.md:854]: Sorry, I do not understand what that means. Can you rephrase or put some details? So it is about aggregation of certain values over time? What is the `wikibaseCount`?

Comment (rti) [reply docs/data-list.md:854]: Ah, now I got it. So it aggregates the creation dates. So the output above shows how many wikibase instances have been created in each year. Is that correct?

Comment (RickiJay-WMDE) [reply docs/data-list.md:854]: Exactly

File: docs/data-list.md [hunk 2]
```diff
@@ -715,13 +810,54 @@ Data abbreviated for brevity.
 
 Data aggregated from the `mostRecent` record for each Wikibase.
 
-All data is paginated as above; the Page Number and Page Size arguments are the same, and the queries return Meta and Data as above.
+### Aggregated Created
+
+This aggregates the number of wikibases created by year. The creation date is derived from the first log date of each wikibase,.
```
Comment (rti) [inline docs/data-list.md:815]:
```suggestion
This aggregates the number of wikibases created by year. The creation date is derived from the first log date of each wikibase.
```
